src/main/java/org/unibl/etf/epraksa/model/entities/json/OpinionByMentorJSON.java

```diff
@@ -3,7 +3,7 @@
 import lombok.Data;
 
 @Data
-public class OpinionByMentor {
+public class OpinionByMentorJSON {
     private Long id;
```

Comment:
dane stavi da su int-ovi, a from i to da su LocalDate

Reply:
broj sati??